frontend/src/components/section/OptionSettings/OptionSettings.styled.ts

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+import { CSSProperties } from 'react';
+
+export const Container = styled.div`
+  width: 100%;
+`;
+
+export const Wrapper = styled.div<CSSProperties>`
+  width: 100%;
+
```

Comment:
제가 알기로는 CSSProperties란 스타일 객체 자체의 타입을 이야기할 때 쓰는 것으로 알고 있습니다! 즉, inline으로 넣는 객체의 타입으로 알고 있습니당 🤔
제가 판단했을 때 해삐의 Wrapper의 의도는 props로 값을 받아서 지정하는 것으로 보이는데 이 부분을 interface로 빼는 것은 어떻게 생각하시나요? 혹시 CSSProperties로 지정한 이유가 있으시다면 그 부분도 궁금합니다 !

```suggestion
interface FlexProps {
  flexDirection?: string;
  justifyContent?: string;
  alignItems?: string;
  gap?: string;
  padding?: string;
  border?: string;
  borderRadius?: string;
}

export const Wrapper = styled.div<FlexProps>`
  width: 100%;
  display: flex;
  flex-direction: ${({ flexDirection = 'row' }) => flexDirection};
  justify-content: ${({ justifyContent = 'start' }) => justifyContent};
  align-items: ${({ alignItems = 'stretch' }) => alignItems};
  gap: ${({ gap = '0px' }) => gap};
  padding: ${({ padding = '0' }) => padding};
  border: ${({ border = 'none' }) => border};
  border-radius: ${({ borderRadius = '0rem' }) => borderRadius};
`;

```

Reply:
제가 CSSProperties를 사용해본 건 interface로 작성하면 지정할 스타일이 늘어날 때마다 수동으로 추가해주어야 하는 번거로움 때문에 사용해본 것이었어요! 타입으로 바로 사용해도 된다고 생각했는데 다시 찾아보니까 스타일 객체의 타입에 대해서 사용하는 용도였다는 걸 알았어요,,🥲 타입 문제가 발생할 수도 있고 해서 interface 방식으로 수정하는 게 더 좋을 것 같네요!

Reply:
동의합니다!